swarm/network/stream/lightnode_test.go

```diff
@@ -21,95 +21,11 @@ import (
 	p2ptest "github.com/ethereum/go-ethereum/p2p/testing"
 )
 
-// This test checks the default behavior of the server, that is
-// when it is serving Retrieve requests.
-func TestLigthnodeRetrieveRequestWithRetrieve(t *testing.T) {
-	registryOptions := &RegistryOptions{
-		Retrieval: RetrievalClientOnly,
-		Syncing:   SyncingDisabled,
-	}
-	tester, _, _, teardown, err := newStreamerTester(registryOptions)
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer teardown()
-
-	node := tester.Nodes[0]
-
-	stream := NewStream(swarmChunkServerStreamName, "", false)
-
-	err = tester.TestExchanges(p2ptest.Exchange{
-		Label: "SubscribeMsg",
-		Triggers: []p2ptest.Trigger{
-			{
-				Code: 4,
-				Msg: &SubscribeMsg{
-					Stream: stream,
-				},
-				Peer: node.ID(),
-			},
-		},
-	})
-	if err != nil {
-		t.Fatalf("Got %v", err)
-	}
-
-	err = tester.TestDisconnected(&p2ptest.Disconnect{Peer: node.ID()})
-	if err == nil || err.Error() != "timed out waiting for peers to disconnect" {
-		t.Fatalf("Expected no disconnect, got %v", err)
-	}
-}
-
-// This test checks the Lightnode behavior of server, when serving Retrieve
-// requests are disabled
-func TestLigthnodeRetrieveRequestWithoutRetrieve(t *testing.T) {
```

Comment:
not sure what this is testing without testing either

Reply:
it seems to test that if `RETRIEVE_REQUEST` stream is not registered, then we should get an error. Not sure how that is valuable at all, since I've never see any actual usage of `RETRIEVE_REQUEST` stream.

I also don't see any light node tests other than these, so it is kinda hard to say if we are breaking anything or not.